src/partition/repart.c

```diff
@@ -3940,6 +3980,11 @@ static int partition_format_verity_hash(
                 node = partition_target_path(t);
         }
 
+        if (p->verity_data_block_size == UINT64_MAX)
+                p->verity_data_block_size = context->fs_sector_size;
+        if (p->verity_hash_block_size == UINT64_MAX)
+                p->verity_hash_block_size = context->fs_sector_size;
```

Comment:
Actually this doesn't work, `fs_sector_size` isn't yet initialized properly by the time we read the configuration. Please move this logic to `partition_format_verity_hash()`

Reply:
I know it isn't obvious from the patch, but those changes are in `partition_format_verity_hash()`.

Reply:
Ah oops I didn't notice